src/main/markdown/makinggwtbetter.md

```diff
@@ -564,22 +564,10 @@ you can clone the repository:
 
 `git clone https://github.com/gwtproject/gwt-site`
 
-The source code for this webpage will be found in *src/main/markdown*. You can edit existing files and add new ones. If you want to see your changes you will need to run:
-
-`mvn clean install`
-
-After that you can go to *target/generated-site* and see the generated site.
-
-You can also launch the embedded Jetty server with
-
-```
-mvn jetty:run
-```
-
-and point your browser to [http://localhost:8080](http://localhost:8080).
+The source code for this webpage will be found in *src/main/markdown*. You can edit existing files and add new ones. For [building](https://github.com/gwtproject/gwt-site#building) and [running](https://github.com/gwtproject/gwt-site#running-locally) the pages see https://github.com/gwtproject/gwt-site#building
```

Comment:
How about simply saying that "instructions to test the changes locally can be found in the README"?